website/src/app/portal/components/data-table/elements/action-cell.tsx

```
@@ -1,67 +1,12 @@
-export function ActionCell<TData, TValue>({ ctx, readOnly = false }: ActionCellProps<TData, TValue>) {
-	const onEdit = () => {
-		if (readOnly) return;
-		console.log('Edit action for row:', ctx.row.original);
-	};
-
-	const onDelete = () => {
-		if (readOnly) return;
-		console.log('Delete action for row:', ctx.row.original);
-	};
-
+export function ActionCell<TData, TValue>({}: CellType<TData, TValue>) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)